src/battleground/HealthBar.js

```diff
@@ -0,0 +1,47 @@
+//@flow strict
+
+import * as React from 'react';
+import "../Main.css";
+import Tank from '../tanks/Tank.js';
+
+type Props = {
+	playersTank?: Tank,
```

Comment:
Is there a case where battleground won't have a tank?

Reply:
Thats when the battleground still hasn't returned the tank